package.json

```diff
@@ -1,9 +1,9 @@
 {
-  "name": "@demos-europe/demosplan-ui",
-  "version": "0.0.2",
+  "name": "my-webpack-project",
+  "version": "1.0.0",
```

Comment:
We might want to roll back some changes within here?

Reply:
I agree, I have overseen this part. But I am afraid I will need help to rollback that file properly.

Reply:
I only meant to restore the following fields to the values they had before:

```
"name": "@demos-europe/demosplan-ui",
"version": "0.0.2",
"description": "Vue components, Vue directives, Design Token and Scss files to build interfaces for demosPlan."
```